asapdiscovery-alchemy/asapdiscovery/alchemy/cli/prep.py

```diff
@@ -219,6 +219,7 @@ def alchemize(
     "--dataset-name",
     type=click.STRING,
     help="The name of the AlchemyDataset this will also be the name of the folder created.",
+    required=True,
```

Comment:
Is this needed?

Reply:
not from this PR, but reminds me that I've wanted to change this for a while. Changing to default "production".